airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -40,6 +40,18 @@
 # we got this when provided API Token has incorrect format
 CLOUDFLARE_ORIGIN_DNS_ERROR = 530
 
+VALID_JSON_SCHEMA_TYPES = {"string", "integer", "number", "boolean", "object", "array", "bit"}
```

Comment:
What is bit type? Cant find it definition for json-schema https://json-schema.org/understanding-json-schema/reference/type.html

Reply:
It's supported by airbyte https://docs.airbyte.io/understanding-airbyte/basic-normalization
Anyway, I'll remove it for a better interoperability. Thanks